frontend/src/app/main/ui/workspace/sidebar/options/rows/color_row.scss

```diff
@@ -34,156 +39,289 @@
   --detach-icon-foreground-color: none;
 
   display: grid;
-  flex: 1;
   grid-template-columns: 1fr auto;
   align-items: center;
-  gap: deprecated.$s-2;
-  border-radius: deprecated.$s-8;
-  background-color: var(--input-details-color);
-  height: deprecated.$s-32;
+  flex: 1;
+  gap: var(--sp-xxs);
+  height: $sz-32;
```

Comment:
Not required for vertical properties (with the Japan thing I am not so sure anymore) but logical properties are cooler.